src/conf.py

```diff
@@ -27,6 +28,7 @@
     "ferrocene_spec",
     "ferrocene_spec_lints",
     "ferrocene_toctrees",
+    "ferrocene_theme",
```

Comment:
Themes don't have to be defined in packages, Ferrocene only did that to be able to distribute it on PyPI. If you move `exts/ferrocene_theme` to `themes/ferrocene` and add `html_theme_path = ["../themes"]` to `conf.py`, loading the theme will still work (but without an extension in the middle).